e2e/nomostest/wait-for-sync.go

```diff
@@ -0,0 +1,482 @@
+// Copyright 2022 Google LLC
```

Comment:
Naming convention for files is snake case, right? Ditto with `config-sync.go`

Reply:
config-sync.go was already there. I just followed the package precedence, but I can rename them both.

Reply:
done